sda-commons-server-auth/src/main/java/org/sdase/commons/server/opa/OpaBundle.java

```diff
@@ -206,6 +251,30 @@ private Builder(OpaConfigProvider<C> opaConfigProvider) {
       this.opaConfigProvider = opaConfigProvider;
     }
 
+    @Override
+    public <T extends Configuration> OpaExtensionsBuilder<T> withOpaConfigProvider(
+        OpaConfigProvider<T> opaConfigProvider) {
+      return new Builder<>(opaConfigProvider);
+    }
+
+    @Override
+    public OpaExtensionsBuilder<C> withoutHeadersExtension() {
+      this.addHeadersExtension = false;
+      return this;
+    }
+
+    @Override
+    public OpaBuilder<C> withInputExtension(OpaInputExtension extension) {
+      // Check if the namespace of an extension would override any original field of the OpaInput.
+      if (Arrays.stream(OpaInput.class.getDeclaredFields())
```

Comment:
You care about the known properties but what is about already added `OpaInputExtension`s?

Reply:
I will also check that there are no conflicts with already registered namespaces.